maintenance/wikia/setWikiFactoryVariable.php

```diff
@@ -82,7 +82,7 @@ public function execute() {
 		}
 
 		if ( !empty( $wikiId ) ) {
-			$wikiIds = [ $wikiId ];
+			$wikiIds = explode(',', $wikiId);
```

Comment:
Nice :) I thought initially we'll need more changes in the maintenance script :+1: